rllib/examples/connectors/connector_v2_frame_stacking.py

```diff
@@ -143,18 +143,27 @@ def _env_creator(cfg):
             lr=0.00015 * (args.num_gpus or 1),
             grad_clip=100.0,
             grad_clip_by="global_norm",
-            model=dict(
-                {
-                    "vf_share_layers": True,
-                    "conv_filters": [[16, 4, 2], [32, 4, 2], [64, 4, 2], [128, 4, 2]],
-                    "conv_activation": "relu",
-                    "post_fcnet_hiddens": [256],
-                },
-                **({"uses_new_env_runners": True} if args.enable_new_api_stack else {}),
-            ),
         )
     )
-
+    if args.enable_new_api_stack:
```

Comment:
I guess we could simplify this, but it does have a benefit to leave things this explicit.

Leave as-is.